<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tcpClientFileSend</class>
 <widget class="QMainWindow" name="tcpClientFileSend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>桌面传输系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>441</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QLabel" name="clientStatusLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>客户端就绪</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>断开</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>107</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditHost">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>20</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>label</zorder>
   <zorder>startButton</zorder>
   <zorder>openButton</zorder>
   <zorder>clientStatusLabel</zorder>
   <zorder>lineEditHost</zorder>
   <zorder>disconnectButton</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
